plumber_rest_api/README.md

```diff
@@ -0,0 +1,102 @@
+# Plumber REST API as a DataStorage provider for shiny.telemetry
+
+> REST API that receives write and read requests for telemetry data from a Shiny Dashboard _(using `shiny.telemetry` R 📦)_
+
+This [plumber](https://www.rplumber.io/) REST API serves as a middleware between a [Shiny](https://shiny.rstudio.com/) Application that is being monitored by `shiny.telemetry` and a data backend that write the logging information to persistent storage. 
+
+The data backend is provided by the `shiny.telemetry` R package.
+
+Features:
+
+* Supports all data storage backends available to `shiny.telemetry`
+    * Writes and reads information
```

Comment:
I think there should be no leading whitespaces here.

Reply:
It was meant to have the whitespace so it's a sub-bullet point, but it adds nothing and I'll remove it.